### PR TITLE
Added Krita to open hardware

### PR DESCRIPTION
**Pull request by Caitlin-Fotheringham**
https://invent.kde.org/graphics/krita

Krita is an open-source, free design application tool, compared against Photoshop for being the best free version of similar software.

## Summary:

Added:

- one
- two
- three

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 ### Open Hardware
 * [Autodesk Circuits](https://circuits.io/) A web-based Arduino simulator
 * [S4A](http://s4a.cat/) Based on Scratch, learn, teach your kids Arduino
+* [Krita](https://krita.org/en/) An end-to-end solution for creating digital art files
```
Comment (Fcmam5) [inline README.md:140]: Wrong category, please create a new design category after `Extensions for Text editors` and theb add Krita,

**Review by Fcmam5 — CHANGES_REQUESTED** [review]

**Comment by Caitlin-Fotheringham** [comment]
I have created the category, I noticed Extensions for Text Editors is not linked at the top in the menu, so wasn't sure whether to add design or not to the menu as well. Do you want me to add it to the menu?

**Comment by oussamabonnor1** [comment]
That would be lovely, yes. Thanks!

**Review by oussamabonnor1 — APPROVED** [review]
Thank you for your contribution!

**Review by Fcmam5 — APPROVED** [review]

**Pull request merged**